setup.py

```diff
@@ -29,6 +29,9 @@
         'simplejson',
         'six>=1.4.0',
     ],
+    extras_require={
+        'uwsgi': ['uWSGI>=2.0.14'],
```

Comment:
In theory the mule return value was backported to all older version, but it's worth verifying - I'm not certain exactly what is entailed in that process for uWSGI.